ci/cudf_pandas_scripts/pandas-tests/job-summary.py

```diff
@@ -68,8 +68,20 @@ def emoji_failed(x):
 pr_df = pd.DataFrame.from_dict(pr_results, orient="index").sort_index()
 main_df = pd.DataFrame.from_dict(main_results, orient="index").sort_index()
 diff_df = pr_df - main_df
+pr_df['CPU Usage'] = ((pr_df['_slow_function_call']/(pr_df['_slow_function_call'] + pr_df['_fast_function_call']))*100.0).round(1)
```

Comment:
Nit: Could you put the denominator calculation on it's own line?

Reply:
Done